polaris-react/package.json

```diff
@@ -72,10 +72,6 @@
   "devDependencies": {
     "@babel/core": "^7.15.0",
     "@babel/node": "^7.14.9",
-    "@rollup/plugin-image": "^2.0.5",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-replace": "^2.3.3",
-    "@rollup/pluginutils": "^3.1.0",
```

Comment:
Major version bump here, We should confirm that this doesn't have an adverse effect on build output (and it might result in a yarn.lock change)

Reply:
Ran build two times and the diff was the same.